<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editcompanywindow</class>
 <widget class="QDialog" name="editcompanywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Company</string>
  </property>
  <widget class="QLabel" name="editCompanyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit Your Company:</string>
   </property>
  </widget>
  <widget class="QLabel" name="companyNameLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>companyName</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="editCompanyTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>661</width>
     <height>701</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>661</width>
     <height>705</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="HireEmployee">
    <attribute name="title">
     <string>Hire Employee(s)</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="employeeNameLineEdit">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="employeePhoneLineEdit">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="employeeNameLabel">
        <property name="font">
         <font>
          <family>Gill Sans MT</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Names: (comma separated: a,b,c..)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="employeePhoneLabel">
        <property name="font">
         <font>
          <family>Gill Sans MT</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Phone Numbers: (comma separated)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="HireEmployeeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Add one or more employees:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HireButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Hire Employee(s)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employeeReportLineEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>377</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeeReportLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>194</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Employees report to: </string>
     </property>
    </widget>
    <widget class="QLabel" name="hirePromptLabel">
     <property name="geometry">
      <rect>
       <x>308</x>
       <y>190</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>prompt</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="FireEmployee">
    <attribute name="title">
     <string>Fire or Replace Employee</string>
    </attribute>
    <widget class="QLabel" name="employeeSearchLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Search Employee to Fire or Replace:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employeeSearchLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>377</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeeSearchNameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmployeeSearchButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="chooseFireOptionLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Choose Option:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="replaceEmployeeRadioButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Fire Employee and Replace with New Employee</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fireEmployeeRadioButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Fire Employee</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="firePromoteOneRadioButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>511</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Fire Employee, and Promote One Employee Supervised Under him/her</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="firePromoteAllRadioButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>491</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Fire Employee, and Promote All Employees Supervised Under him/her</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmployeeReplaceButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>300</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeeSearchPrompt">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>prompt</string>
     </property>
    </widget>
    <widget class="QLabel" name="fireOptionPrompt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>481</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Prompt</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeeModNameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newEmployeeNameLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>377</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newEmployeePhoneLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>377</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeeModPhoneLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Phone:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FirePromoteAllButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>130</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Proceed</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="PromoteEmployee">
    <attribute name="title">
     <string>Change Boss</string>
    </attribute>
    <widget class="QLabel" name="employeePromoteSearchLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Search Employee to promote or demote:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmployeeFindBossButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeePSearchNameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employeePromoteSearchLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>361</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeeReportsToLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Employee currently reports to:</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeePromoteSearchPrompt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>prompt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employeeNewBossLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>361</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeePSearchNameLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>New Boss:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmployeeChangeBossButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>150</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Change Boss</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="EditEmployee">
    <attribute name="title">
     <string>Edit Employee</string>
    </attribute>
    <widget class="QLabel" name="employeeESearchLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Search Employee:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmployeeESearchButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeeESearchNameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employeeESearchLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>361</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeeESearchPrompt">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>prompt</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeeModPhoneLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>New Phone:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmployeeEditButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>143</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employeeNewPhoneLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>361</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employeeNewNameLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>361</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeeModNameLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>New Name:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="UpdateCompanyButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Company View</string>
   </property>
  </widget>
  <widget class="QLabel" name="CompanyViewLabel">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>50</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Company Tree:</string>
   </property>
  </widget>
  <widget class="QLabel" name="CompanyDisplay">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>691</width>
     <height>671</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>671</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>751</width>
     <height>671</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
